nextflow.config

```diff
@@ -43,6 +43,8 @@ params {
     amp_hmmsearch_savetargets       = false
     amp_hmmsearch_savedomains       = false
 
+    amp_tool_dict                   = '{"ampir":"ampir.tsv", "amplify":".tsv", "macrel":".prediction", "neubi":"neubi.fasta", "hmmer_hmmsearch":".txt"}'
```

Comment:
Is `neubi` correct? We don't support it do we? Or Does it not matter?

Reply:
Ok we hardcoded it in now in the modules.config